README.md

```diff
@@ -23,6 +23,53 @@ providing an insightful and convenient interface to the Open Source project data
 
 ---
 
+## Contributing
+
+Please see our guide to contributing to this project at the following site: [CONTRIBUTORS.MD](https://github.com/oss-aspen/8Knot/blob/dev/docs/CONTRIBUTORS.md)
+
+The file-structure of our application is intuitive. Non-application files are omitted from this overview:
```

Comment:
I think the file structure should be its own sections vs being under contributing

Reply:
okay that's fair

Reply:
done